mne/io/nihon/nihon.py

```diff
@@ -23,7 +23,7 @@ def _ensure_path(fname):
 
 
 @fill_doc
-def read_raw_nihon(fname, preload=False, verbose=None) -> "RawNihon":
+def read_raw_nihon(fname, preload=False, encoding="utf-8", verbose=None) -> "RawNihon":
```

Comment:
Must be added before `verbose` in case someone used it as a `arg` (which they shouldn't have anyway really...) and we're trying to add `*`s anyway, so:
```suggestion
def read_raw_nihon(fname, preload=False, *, encoding="utf-8", verbose=None) -> "RawNihon":
```